### PR TITLE
Styled TextFieldWithLabel

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://trello.com/c/95Nr1LgN/94-migrate-composer-forms-to-styled-components

### CODE REVIEW

File: packages/styled-components/src/components/primitives/Form/TextFieldWithLabel/TextFieldWithLabel.test.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  });
+  it('should render label', () => {
+    givenComponentRendered({ label: 'foo' });
+    screen.getByText('foo');
+  });
+  it('should render inputMode', () => {
+    givenComponentRendered({ inputMode: TextFieldWithLabel.Modes.Numeric });
+    const textFieldWithLabel = screen.getByTestId(testId);
+    expect(textFieldWithLabel).toHaveAttribute('inputmode', 'numeric');
+  });
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/TextFieldWithLabel/TextFieldWithLabel.test.tsx:62]: missing unit tests for both custom css properties

File: packages/styled-components/src/components/primitives/Typography/Typography.tsx [hunk 2]
```diff
@@ -12,8 +12,10 @@ export const Typography = ({
   mode,
   htmlFor,
   testId = 'typography',
+  customCss,
 }: Props) => (
   <StyledTypography
+    css={customCss}
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Typography/Typography.tsx:18]: missing unit test

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/styled-components/src/components/primitives/Typography/Typography.tsx (hunk 2 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Typography/Typography.tsx:18]: added

File: packages/styled-components/src/components/primitives/Form/TextFieldWithLabel/TextFieldWithLabel.test.tsx (hunk 1 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/TextFieldWithLabel/TextFieldWithLabel.test.tsx:63]: added

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**